app/data/schema/schema-v1.json

```diff
@@ -17,7 +17,8 @@
                     "enum": [
                         "equals",
                         "not equals",
-                        "contains"
+                        "contains",
+                        "unanswered"
```

Comment:
Would "not set" be better? Just thinking if it's a metadata rule then it might not be an answer that's being checked against.